### ISSUE TITLE
OAEP encryption using SHA1?

### ISSUE THREAD

**Issue by core433** [crypto-browserify/crypto-browserify#172]
I've posted this issue on the public-encrypt repo as well:

https://github.com/crypto-browserify/publicEncrypt/issues/13

Public Encrypt is using sha1 for the hash function, is that secure? It sounds like sha1 is being deprecated because it's been theoretically crackable since 2005. Is this the same hash function that node crypto is using / are they not on sha2 / sha256?

**Comment by dominictarr** [comment]
yes probably but the goal here is to be fully compatible with the node crypto api, so we'll remove this when they do.
